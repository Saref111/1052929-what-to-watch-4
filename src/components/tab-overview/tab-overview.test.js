import React from "react";
import renderer from "react-test-renderer";
import TabOverview from "./tab-overview.jsx";

test(`Snapshot test for TabOverview`, () => {
  const tree = renderer.create(<TabOverview
    rate={2}
    rateString={`2`}
    votes={222}
    description={{
      prescription: `123qweasdxcv`,
      postscription: `123qweasdxcv`
    }}
    director={`qweerttyu`}
    actors={`sdfghjk sdfsdf sdfsdf sdfsdfsdf`}
  />);

  expect(tree).toMatchSnapshot();
});
