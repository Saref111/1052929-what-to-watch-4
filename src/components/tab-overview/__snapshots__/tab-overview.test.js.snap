// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for TabOverview 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      2
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        2
      </span>
      <span
        className="movie-rating__count"
      >
        222
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      [object Object]
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        qweerttyu
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        sdfghjk sdfsdf sdfsdf sdfsdfsdf
         and other.
      </strong>
    </p>
  </div>,
]
`;
